extensions/amp-story-interactive/0.1/amp-story-interactive-slider.css

```diff
@@ -13,3 +13,37 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+.i-amphtml-story-interactive-slider-input-container {
+  background: var(--i-amphtml-interactive-option-background-color) !important;
+  border-radius: 2em !important;
```

Comment:
We need to specify this to only apply to the top corners.
Otherwise it's round on the bottom corners
<img width="80" alt="Screen Shot 2021-07-01 at 3 41 34 PM" src="https://user-images.githubusercontent.com/3860311/124180961-e6efa200-da82-11eb-906b-07f4082ebc42.png">